bucket/clash-for-windows.json

```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://github.com/Fndroid/clash_for_windows_pkg",
+    "description": "A Windows/macOS GUI based on Clash",
+    "version": "0.13.1",
+    "license": "MIT",
+    "url": "https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.13.1/Clash.for.Windows.Setup.0.13.1.exe#/clash.7z",
+    "hash": "8ff7c72573fb8c2c620d815e6d7531825f7bd8fdfcae0fe135b7825150f6fdf9",
+    "shortcuts": [
+        [
+            "Clash for Windows.exe",
+            "Clash"
+        ]
+    ],
+    "pre_install": [
+        "Remove-Item -Recurse \"$dir\\Uninstall Clash for Windows.exe\"",
+        "Get-ChildItem -Path \"$dir\\`$PLUGINSDIR\" -Depth \"2\" -Include \"*.7z\" | Move-Item -Destination \"$dir\\`$PLUGINSDIR\"",
+        "Expand-7zipArchive -Path \"$dir\\`$PLUGINSDIR\\app-64.7z\" -DestinationPath \"$dir\"",
+        "Remove-Item -Recurse \"$dir\\`$PLUGINSDIR\""
+    ],
+    "persist": "data",
```

Comment:
> The persist is to support the new portable mode according to the doc of cfw,

What docs?? Where are these docs?

Reply:
> > The persist is to support the new portable mode according to the doc of cfw,
> 
> What docs?? Where are these docs?

Emm, it's a doc written by the author of the app with gitbook. https://docs.cfw.lbyczf.com/

Reply:
https://github.com/lukesampson/scoop-extras/pull/5049#discussion_r517188994

Reply:
> [#5049 (comment)](https://github.com/lukesampson/scoop-extras/pull/5049#discussion_r517188994) Just close the PR and move to jp buckets

OK